s3torchconnectorclient/python/tst/integration/test_mountpoint_s3_integration.py

```diff
@@ -258,6 +258,40 @@ def test_head_object(sample_directory):
         assert object_info.etag == expected_etag
 
 
+@pytest.mark.parametrize("delete_calls", [1, 2])
+def test_delete_object(delete_calls: int, sample_directory):
+    client = MountpointS3Client(sample_directory.region, TEST_USER_AGENT_PREFIX)
+    for i in range(delete_calls):
```

Comment:
Could you please explain why do we need to call delete_object twice here?

Reply:
The idea was to verify that `delete_object` does not throw in case the object is already deleted, but it's also tested in `test_delete_object_does_not_exist`, so I'm happy to remove it here